### DISCUSSION TITLE
Doris FE高可用疑问

### DISCUSSION THREAD

**Discussion by ixiaoyi93** [apache/doris#35168] [A - General / Q&A]
FE是多个Follower进行选举出一个Master，通过官方文档得知 Web管理台可以通过 Nginx 代理实现高可用。在这里我有以下几个疑问：

- BE在创建之后或之前注册到FE中，在注册的时候需要指定 FE Master 地址。那么如果 FE Follower 发送高可用切换是否应用现有节点与 新选举的 Master 节点的通讯呢？
- 如果FE Follower故障选举出新的Master节点，后续我在添加节点是指定新的Master 节点地址？
- 如果FE Follower故障选举出新的Master节点，应用程序或可视化数据库工具，应该使用那个地址来链接 FE master ?

还请官方大佬们给出一些建议和提示，感谢。

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
